src/System.Private.CoreLib/src/System/Runtime/Serialization/SerializationInfo.cs

```diff
@@ -2,46 +2,28 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-/*============================================================
-**
-**
-**
-** Purpose: The structure for holding all of the data needed
-**          for object serialization and deserialization.
-**
-**
-===========================================================*/
-
-using System;
 using System.Collections.Generic;
-using System.Reflection;
-using System.Globalization;
 using System.Diagnostics;
-using System.Security;
-using System.Runtime.CompilerServices;
 
 namespace System.Runtime.Serialization
 {
+    /// <summary>The structure for holding all of the data needed for object serialization and deserialization.</summary>
     public sealed class SerializationInfo
     {
-        private const int defaultSize = 4;
-        private const string s_mscorlibAssemblySimpleName = System.CoreLib.Name;
-        private const string s_mscorlibFileName = s_mscorlibAssemblySimpleName + ".dll";
+        private const int DefaultSize = 4;
 
         // Even though we have a dictionary, we're still keeping all the arrays around for back-compat. 
         // Otherwise we may run into potentially breaking behaviors like GetEnumerator() not returning entries in the same order they were added.
-        internal String[] m_members;
-        internal Object[] m_data;
-        internal Type[] m_types;
-        private Dictionary<string, int> m_nameToIndex;
-        internal int m_currMember;
-        internal IFormatterConverter m_converter;
-        private String m_fullTypeName;
-        private String m_assemName;
-        private Type objectType;
-        private bool isFullTypeNameSetExplicit;
-        private bool isAssemblyNameSetExplicit;
-        private bool requireSameTokenInPartialTrust;
+        private string[] _names;
```

Comment:
@jkotas do you know why some of these were internal?

Reply:
No idea. It is not unusual for the internal/private in the old code like this to be non-nonsensical.